e2e_tests/src/_utils/logger.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs');
+
+const Colors = {
+    FgRed: "\x1b[31m",
+    FgGreen: "\x1b[32m",
+    FgYellow: "\x1b[33m",
+    FgBlue: "\x1b[34m",
+    FgMagenta: "\x1b[35m",
+    FgWhite: "\x1b[37m",
+    RESET: "\x1b[0m"
+}
+
+const output_dir = `${__dirname}/../../states/logger`
+
+class logger {
+
+    /**
+     * Save results of a test
+     * @param test_path
+     * @param name
+     * @param results
+     * @returns {Promise<void>}
+     */
+    static save = (test_path, name, results) => {
+        const splited = test_path.split('/')
```

Comment:
Great job on the PR @ali-fs 
just a variable name change otherwise it LGTM.
```suggestion
        const split = test_path.split('/')
```

Reply:
Thanks @matin-binary. I'll change this.